.changeset/popular-shoes-carry.md

```diff
@@ -0,0 +1,6 @@
+---
+'lit-html': patch
+'lit': patch
+---
+
+[lit-html] styleMap fails when update contains priority directive
```

Comment:
Instead of writing the description of the bug, write the description of the fix. And don't use a package prefix, that's already in the metadata:

```suggestion
Fix styleMap's handling of important flags
```